app/models/miq_server/environment_management.rb

```diff
@@ -29,7 +29,7 @@ def minimal_env_options
         options.shift # remove the "minimal" from the front of the array
 
         # Special case where Netbeans is handling the UI worker for debugging
-        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting noui) : o }.flatten
+        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting no_ui) : o }.flatten
```

Comment:
Yep, my bad, misunderstood.  We really just need a way to run the workers without starting the ui, so `no_ui` should be fine and we no longer need `netbeans`.  This would cover the use case where we need minimal workers running, say `reporting` and one `generic` to support the UI running in an IDE session.